src/sgame/sg_cmds.cpp

```diff
@@ -4442,6 +4451,58 @@ void Cmd_PrivateMessage_f( gentity_t *ent )
 	}
 }
 
+
+/*
+=================
+Cmd_ReplyPrivateMessage_f
+
+Reply by private message to the last private message sender
+=================
+*/
+
+void Cmd_ReplyPrivateMessage_f(gentity_t *ent)
+{
+	char *msg;
+	int target;
+	bool teamonly = false;
+
+	if (!g_privateMessages.Get() && ent)
+	{
+		ADMP( "\"" N_("Sorry, but private messages have been disabled") "\"" );
+		return;
+	}
+
+	if (trap_Argc() < 2)
+	{
+	ADMP( QQ( N_("usage: /r [message]") ) );
+	return;
+	}
+
+	if ( level.time - ent->client->pers.lastPrivateMessageSenderTime < 3000 )
+	{
+		ADMP( "\"" N_("More than one possible recipient, refusing to send.") "\"" );
+        return;
+	}
+
+	target = ent->client->pers.lastPrivateMessageSender;
+	if (target == -1 || !g_entities[target].client)
+	{
+		ADMP( "\"" N_("No one to reply to.") "\"" );
+		return;
+	}
+
+	msg = ConcatArgs(1);
+
+	if (G_SayTo(ent, &g_entities[target], teamonly ? SAY_TPRIVMSG : SAY_PRIVMSG, msg))
```

Comment:
Let's also print who we are replying to

Reply:
To be clear, I mean send a message to the person who sent the command:

Something like (didn't test):
```
ADMP( va( "%s %s", QQ( N_("Sent message to %s") ), g_entities[target].client->pers.netname ) );
```

Reply:
Yeah. Name needs to be quoted though

Reply:
We are doing that. Well are doing it like it do with `/m`

[Sender] => [Target] : [message] 
Like this : 

![91_1](https://github.com/user-attachments/assets/38883511-397e-486b-a1ab-86267cd83dbb)

Reply:
No, that message is generated client-side for the recipient of the message. What we're saying is that the sender of the message should see something.

Reply:
I don't say that I will do exactly that.
I will do something like this (It will display something like this)

Reply:
Done